packages/beacon-node/src/api/rest/swaggerUI.ts

```diff
@@ -43,7 +43,7 @@ async function getAsset(name: string): Promise<Buffer | undefined> {
     const url = await import("node:url");
     // eslint-disable-next-line @typescript-eslint/naming-convention
     const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
-    return await fs.readFile(path.join(__dirname, "../../../../../assets/", name));
+    return await fs.readFile(path.join(__dirname, "../../../../../docs/assets/", name));
```

Comment:
When i originally moved the assets into the docs folder I didn't realize that they were being used by the api.  Should they get moved back out to the root?  I was trying to tidy up the repo by nesting that extra folder in with docs.  What do you all think?

Reply:
If assets are not strictly used in docs I think it's better to move them out, makes it safe to just exclude whole docs folder in `.dockerignore`. Assets might also be something that 3rd parties use (like dappnode), having that folder in root might be a good thing for visibility.